druid/src/test/java/org/apache/calcite/test/DruidAdapterIT2.java

```diff
@@ -3928,15 +3927,13 @@ public void testExtractQuarter() {
 
   @Test
   public void testCastTimestamp() {
-    final String sql = "Select cast(cast(\"timestamp\" as timestamp) as varchar) as t"
+    final String sql = "Select cast(\"timestamp\" as varchar) as t"
```

Comment:
@danny0405 - I wonder if we should leave the previous test case as is (or you can create new one for `cast("timestamp" as varchar)`). The previous one `cast(cast("timestamp" as timestamp) as varchar)` was added in https://github.com/apache/calcite/commit/8a91c8dd4510f0741242f89c1a988e46036a2139 for solving issue [CALCITE-2387](https://issues.apache.org/jira/browse/CALCITE-2387), but it was no longer working after some changes between release 1.17 and 1.18. It would be great if we see this fix makes the case work again.

Reply:
@zhztheplayer I have added the case in the JIRA, these 2 cases are all fixed. Thx for the review,